src/Apps/Artist/Routes/Overview/__test__/ArtworkFilter.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { FilterState } from "Apps/Artist/Routes/Overview/state"
+import React from "react"
+import { graphql } from "react-relay"
+import { Provider } from "unstated"
+import {
+  MockBoot,
+  MockRelayRenderer,
+  renderRelayTree,
+  renderUntil,
+} from "../../../../../DevTools"
+import { ArtworkFilterFixture } from "../../../../__test__/Fixtures/Artist/ArtworkFilter"
+import { ArtworkFilterFragmentContainer as ArtworkFilter } from "../Components/ArtworkFilter"
+
+jest.unmock("react-relay")
+
+xdescribe("ArtistHeader", () => {
+  // const getWrapper = async (response = ArtworkFilterFixture) => {
+  //   return await renderRelayTree({
+  //     Component: () => {
+  //       return (
+  //         <Provider inject={[filterState]}>
+  //           <ArtworkFilter />
+  //         </Provider>
+  //       )
+  //     },
+  //     query: graphql`
+  //       query ArtworkFilter_Test_Query {
+  //         artist(id: "pablo-picasso") {
+  //           ...ArtworkFilter_artist
+  //         }
+  //       }
+  //     `,
+  //     mockResolvers: { Artist: () => {} },
+  //   })
+  // }
```

Comment:
Can you add a `// FIXME: Uncomment tests, currently WIP` or something at the top of the file?

Reply:
Yea, will add, and clean up a bit. Wanted to PR this so I can possibly pair, or merge as a pended spec in the meantime, etc.

Reply:
As long as these WIP sections are clearly labeled i think its ok, but better to just stash changes in a branch until ready